cloud/pkg/synccontroller/objectsync.go

```diff
@@ -50,7 +50,7 @@ func (sctl *SyncController) manageObject(sync *v1alpha1.ObjectSync) {
 		klog.Errorf("failed to get obj(gvr:%v,namespace:%v,name:%v), %v", gvr, sync.Namespace, sync.Spec.ObjectName, err)
 		return
 	}
-
+	filter.MessageFilter(ret, nodeName)
```

Comment:
filter message before sending?
https://github.com/kubeedge/kubeedge/blob/master/cloud/pkg/synccontroller/objectsync.go#L105

Reply:
Yes, when the resources issued for compensation are endpoints or endpointslice, I think filtering should be carried out.